setup.py

```diff
@@ -282,8 +282,8 @@ def finalize_options(self):
     'sphinx',
     'sphinxcontrib-napoleon',
     'plotly',
-    'pattern <= 2.6',
     'sphinxcontrib.programoutput',
+    'annoy',
```

Comment:
why? If I remember correctly, this doesn't work on Mac (or already fixed?)

Reply:
Because of this: https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/similarities/index.py#L50

Reply:
hmm, please check that on gensim-wheels (osx), I guess this can be an issue (and probably better to move that into `__init__` function of indexer, instead of module level